src/mixins/common-mixin.ts

```diff
@@ -51,4 +52,24 @@ export default class CommonMixin extends Vue {
     }
     return true
   }
+
+  /**
+   * Extracts validation flags from validation item detail object
+   * @param validation items details object
+   * @return array of validation flags for all validation items
+   */
+  buildValidFlags (validationItems: ValidationItemDetailIF[]): object {
+    let result = {}
+    for (const vi of validationItems) { result[vi.name] = vi.valid }
+    return result
+  }
+
+  /**
+   * Extracts element ids from validation item detail object
+   * @param validation items details object
+   * @return array of element ids for all validation items
+   */
+  buildElementIds (validationItems: ValidationItemDetailIF[]): object {
```

Comment:
Can return type be `string[]` ?

Reply:
updated to have return type of `string[]`